libs/SmartSync/SmartSync/Classes/Util/SFSyncTarget.m

```diff
@@ -27,6 +27,11 @@
 #import "SFSoqlSyncTarget.h"
 #import "SFSoslSyncTarget.h"
 
+#define ABSTRACT_METHOD {\
```

Comment:
Is this a concept we should generalize to the greater SDK (e.g. `SalesforceSDKConstants.h`)?